src/api/comms.js

```diff
@@ -40,7 +43,8 @@ export function checkStatus(response = {}) {
       case 204:
         return {};
       default:
-        if (response.headers.get('content-type') === 'text/plain') {
+        const contentType = response.headers.get('content-type');
+        if (contentType && contentType.indexOf('text/plain') !== -1) {
```

Comment:
Nit, any advantage to an indexOf and not a contains here? Easier to interpret with contains imo

Reply:
contains was removed from the spec so it would be `includes`, I'll change it 👍 Just too used to having to support IE which doesn't have `includes`